File: codeforces/148A/cpp/148a.cpp
#include <iostream>
using namespace std;

const int c_max = 1e5;

int main()
{
    int k, l, m, n, d;
    cin >> k >> l >> m >> n >> d;

    int num = 0;
    for (int i = 1; i <= d; i++) {
        if (i % k != 0 && i % l != 0 && i % m != 0 && i % n != 0)
            num++;
    }

    cout << (d - num) << endl;
    return 0;
}

File: codeforces/148A/go/148a.go
package main

import "fmt"

func main() {
    var k, l, m, n, d int
    fmt.Scan(&k)
    fmt.Scan(&l)
    fmt.Scan(&m)
    fmt.Scan(&n)
    fmt.Scan(&d)

    num := 0
    for i := 1; i <= d; i++ {
        if i % k != 0 && i % l != 0 && i % m != 0 && i % n != 0 {
            num++
        }
    }

    fmt.Println(d-num)
}
